lib/measured/unit_system_builder.rb

```diff
@@ -8,12 +8,53 @@ def unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def si_unit(unit_name, aliases: [], value: nil)
+    @units += build_si_units(unit_name, aliases: aliases, value: value)
+    nil
+  end
+
   def build
     Measured::UnitSystem.new(@units)
   end
 
   private
 
+  def self.prefixes
+    [
+      ['y', 'yocto', -24],
+      ['z', 'zepto', -21],
+      ['a', 'atto', -18],
+      ['f', 'femto', -15],
+      ['p', 'pico', -12],
+      ['n', 'nano', -9],
+      ['μ', 'micro', -6],
+      ['m', 'milli', -3],
+      ['c', 'centi', -2],
+      ['d', 'deci', -1],
+      ['da', 'deca', 1],
+      ['h', 'hecto', 2],
+      ['k', 'kilo', 3],
+      ['M', 'mega', 6],
+      ['G', 'giga', 9],
+      ['T', 'tera', 12],
+      ['P', 'peta', 15],
+      ['E', 'exa', 18],
+      ['Z', 'zetta', 21],
+      ['Y', 'yotta', 24]
+    ]
+  end
+
+  def build_si_units(name, aliases: [], value: nil)
+    size = value.split(' ')[0] unless !value
+    size ||= 1
+    si_units = [build_unit(name, aliases: aliases, value: value)]
```

Comment:
@thegedge your solution is a bit confusing to me. I was under the impression that the size was not always going to be ```"10e${exp}"```, I thought size was needed to be extracted so it could be multiplied after calculating the exponent. For example, the following case:
```
unit :lb
si_unit :long_ton, value: "2240 lb"
```
Size would need to be extracted, in this case it is ```2240```.
Should this 'feature' be waived so that si_unit only works for units that have a relation of the form ```"10e#{exp}``` ?